packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -328,7 +333,7 @@ export class EditableText extends AbstractPureComponent<IEditableTextProps, IEdi
                 inputWidth: Math.max(scrollWidth, minWidth),
             });
             // synchronizes the ::before pseudo-element's height while editing for Chrome 53
-            if (multiline && this.state.isEditing) {
+            if (multiline && (this.state.isEditing || isControlled)) {
```

Comment:
why make this change here and not also on line 315? seems suspicious.

Reply:
This is the only place where we change parentElement height (in controlled mode we have to do this event with `this.state.isEditing == false`). Line 315 is not related to it